_posts/2023-03-15-shuffling-large-data-at-constant-memory.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: post
+title: Shuffling large data at constant memory in Dask
+author: Hendrik Makait
+tags: [p2p, shuffling, dask, distributed]
+theme: twitter
+canonical_url: https://blog.coiled.io/blog/shuffling-large-data-at-constant-memory.html
+---
+
+{% include JB/setup %}
+
+_Original version of this post appears on [blog.coiled.io](https://blog.coiled.io/blog/shuffling-large-data-at-constant-memory.html?utm_source=dask-blog&utm_medium=shuffling-large-data-at-constant-memory)_
```

Comment:
```suggestion
_This work was engineered and supported by [Coiled](https://coiled.io?utm_source=dask-blog&utm_medium=shuffling-large-data-at-constant-memory). In particular, thanks to Coiled engineers [Hendrik Makait](https://github.com/hendrikmakait), [other person](github-link), and [other person](github-link). Original version of this post appears on [blog.coiled.io](https://blog.coiled.io/blog/shuffling-large-data-at-constant-memory.html?utm_source=dask-blog&utm_medium=shuffling-large-data-at-constant-memory)_
```

@emilyvalentine I'm thinking something like this, thoughts? @jrbourbeau would you mind helping fill in other peoples' names, or be able point me to a PR?

Reply:
I love this idea and the language is great. Thanks @scharlottej13 and @jacobtomlinson !

Reply:
This looks great. Once you have the other engineers' names filled in I'm happy to merge this.

Reply:
Gently pinging @mrocklin and @fjetter on this as they probably know best who was most involved.

Reply:
Thank you @fjetter!